src/sources/util/net/tcp/mod.rs

```diff
@@ -348,6 +348,7 @@ async fn handle_stream<T>(
                             }
                         }
 
+
```

Comment:
Extra linebreak?

Reply:
I don't think that this file adhered to `.editorconfig`'s trailing newline rule.